sdk/python/tests/integration/feature_repos/repo_configuration.py

```diff
@@ -48,7 +48,7 @@
 DYNAMO_CONFIG = {"type": "dynamodb", "region": "us-west-2"}
 # Port 12345 will chosen as default for redis node configuration because Redis Cluster is started off of nodes
 # 6379 -> 6384. This causes conflicts in cli integration tests so we manually keep them separate.
-REDIS_CONFIG = {"type": "redis", "connection_string": "localhost:12345,db=0"}
+REDIS_CONFIG = {"type": "redis", "connection_string": "localhost:6379,db=0"}
```

Comment:
I'm not sure it's safe to do this: Kevin recently changed our integration tests (both on PRs and on master) to start Redis on port 12345 instead of 6379; if you want to change back to 6379, you should change the workflows in `pr_integration_tests.yml` and `master_only.yml` (I would recommend doing that, since Kevin's change was actually unnecessary)

Reply:
btw this is probably the reason all the integration tests are failing

Reply:
but changed it to 12345 and will fix in a different PR.